src/interface/variable_pack.hpp

```diff
@@ -357,6 +409,31 @@ VariablePack<T> MakePack(const vpack_types::VarList<T> &vars,
   }
 }
 
+template <typename T>
+SwarmVariablePack<T> MakeSwarmPack(const vpack_types::SwarmVarList<T> &vars,
+                                   PackIndexMap *vmap = nullptr) {
+  // count up the size
+  int vsize = 0;
+  for (const auto &v : vars) {
+    vsize++;
+  }
+
```

Comment:
Does vars not have a size method? Seems a bit odd.

Reply:
`std::forward_list` actually does not have a `size` method, which seems odd but 🤷